### ISSUE TITLE
Chromium "Aw snap"

### ISSUE THREAD

**Issue by RNieuwenhuis** [genotoul-bioinfo/dgenies#2]
Dear sir/madam,

First, thank you for creating this valuable tool.

I installed it successfully on one of our servers without root access and it works fine.
Several dotplots were created already for genomes up to 1 gbp with both zipped and unzipped fasta as I changed the max sizes in the application.properties file. 

The problem I encounter is that when I use the output of a 10x genomics supernova raw.fasta output the mapping works successfully but upon display chromium fails after a while.

Now I wonder if this is just a timeout because the loading of the plot takes too long. If so, than this seems inherent to using chromium since it is impossible to increase the timeout setting -,-

Or is it the 10x supernova output that is somehow complicating matters?

If it is a timeout issue, would it be possible to implement firefox?

Your views are much appreciated.

Sincerely,

**Comment by florealcab** [comment]
Hi,

Thank you for using D-Genies.

When you launch dgenies, it launch into your default web browser. But you can copy the URL you have in chromium and paste it in firefox. It will work. We support all browsers which follow WEB standards. Chrome/ium and Firefox have been successfully tested both.

You close your issue because your problem is solved? Else we can search which is the problem. Tell us

Regards,

**Comment by RNieuwenhuis** [comment]
Yes it was indeed a timeout by chromium and firefox fixed the issue.

**Comment by florealcab** [comment]
Hum... ok. I should work with chromium too. Can you please tell me the size of the "map.paf" file you have? This file is located into a folder with the name of your job, into the data_folder set in your application.properties file.

**Comment by RNieuwenhuis** [comment]
So in the first place the fastas I used contained a lot of redundant information and should not be used for a dotplot in the first place (my bad)

The filesizes of the map.paf file for all my runs so far:
ls */map.paf | du -h
142M	./GJPeL_20180719182908
224M	./NAZ6Q_20180803185644
192M	./IaPZZ_20180804135907
4.1G	./w1Yq7_20180731105716
489M	./6V2bk_20180719125207
452M	./5qpxD_20180806144504
228M	./g9uQU_20180719184553
418M	./IUGYw_20180719181546
483M	./hsnH2_20180801133823
830M	./BwQmk_20180724114720
472M	./Rncvh_20180719142116
213M	./9jqyi_20180804141357
488M	./YrtIB_20180719162349
483M	./6N35j_20180801115852
123M	./VjJfA_20180719183542
136M	./VjJfA_20180719183542_2

So to summarize: I used a wrong file resulting in a huge mapping that took too long to visualize when checking the results in chromium and therefore it gave a timeout before the plot was displayed. Doing the same in firefox did show the plot.
